entrypoint.sh

```diff
@@ -5,137 +5,121 @@ set -o pipefail
-if [ -z "${SUBSPACE_IPV6_POOL-}" ] ; then
-    export SUBSPACE_IPV6_POOL="fd00::10:97:0/112"
+if [ -z "${SUBSPACE_IPV6_POOL-}" ]; then
+  export SUBSPACE_IPV6_POOL="fd00::10:97:0/112"
 fi
-if [ -z "${SUBSPACE_NAMESERVER-}" ] ; then
-    export SUBSPACE_NAMESERVER="1.1.1.1"
+if [ -z "${SUBSPACE_NAMESERVER-}" ]; then
+  export SUBSPACE_NAMESERVER="1.1.1.1"
 fi
 
-if [ -z "${SUBSPACE_LETSENCRYPT-}" ] ; then
-    export SUBSPACE_LETSENCRYPT="true"
+if [ -z "${SUBSPACE_LETSENCRYPT-}" ]; then
+  export SUBSPACE_LETSENCRYPT="true"
 fi
 
-if [ -z "${SUBSPACE_HTTP_ADDR-}" ] ; then
-    export SUBSPACE_HTTP_ADDR=":80"
+if [ -z "${SUBSPACE_HTTP_ADDR-}" ]; then
+  export SUBSPACE_HTTP_ADDR=":80"
 fi
 
-if [ -z "${SUBSPACE_LISTENPORT-}" ] ; then
-    export SUBSPACE_LISTENPORT="51820"
+if [ -z "${SUBSPACE_LISTENPORT-}" ]; then
+  export SUBSPACE_LISTENPORT="51820"
 fi
 
-if [ -z "${SUBSPACE_HTTP_INSECURE-}" ] ; then
-    export SUBSPACE_HTTP_INSECURE="false"
+if [ -z "${SUBSPACE_HTTP_INSECURE-}" ]; then
+  export SUBSPACE_HTTP_INSECURE="false"
 fi
 
 export DEBIAN_FRONTEND="noninteractive"
 
-if [ -z "${SUBSPACE_IPV4_GW-}" ] ; then
-        export SUBSPACE_IPV4_PREF=$(echo ${SUBSPACE_IPV4_POOL-} | cut -d '/' -f1 |sed 's/.0$/./g' )
-	export SUBSPACE_IPV4_GW=$(echo ${SUBSPACE_IPV4_PREF-}1)
+if [ -z "${SUBSPACE_IPV4_GW-}" ]; then
+  export SUBSPACE_IPV4_PREF=$(echo ${SUBSPACE_IPV4_POOL-} | cut -d '/' -f1 | sed 's/.0$/./g')
+  export SUBSPACE_IPV4_GW=$(echo ${SUBSPACE_IPV4_PREF-}1)
 
 fi
-if [ -z "${SUBSPACE_IPV6_GW-}" ] ; then
-        export SUBSPACE_IPV6_PREF=$(echo ${SUBSPACE_IPV6_POOL-} | cut -d '/' -f1 |sed 's/:0$/:/g' )
-	export SUBSPACE_IPV6_GW=$(echo ${SUBSPACE_IPV6_PREF-}1)
+if [ -z "${SUBSPACE_IPV6_GW-}" ]; then
+  export SUBSPACE_IPV6_PREF=$(echo ${SUBSPACE_IPV6_POOL-} | cut -d '/' -f1 | sed 's/:0$/:/g')
+  export SUBSPACE_IPV6_GW=$(echo ${SUBSPACE_IPV6_PREF-}1)
 fi
 
-if [ -z "${SUBSPACE_IPV6_NAT_ENABLED-}" ] ; then
-    export SUBSPACE_IPV6_NAT_ENABLED=1
+if [ -z "${SUBSPACE_IPV6_NAT_ENABLED-}" ]; then
+  export SUBSPACE_IPV6_NAT_ENABLED=1
 fi
 
 # Set DNS server
 echo "nameserver ${SUBSPACE_NAMESERVER}" >/etc/resolv.conf
 
-if [ -n "${SUBSPACE_DISABLE_MASQUERADE-}" ] ; then
-    # IPv4
-    if  [ -x /sbin/iptables ]; then
-        if ! /sbin/iptables -t nat --check POSTROUTING -s ${SUBSPACE_IPV4_POOL} -j MASQUERADE ; then
-            /sbin/iptables -t nat --append POSTROUTING -s ${SUBSPACE_IPV4_POOL} -j MASQUERADE
-        fi
-
-        if ! /sbin/iptables --check FORWARD -m state --state RELATED,ESTABLISHED -j ACCEPT ; then
-            /sbin/iptables --append FORWARD -m state --state RELATED,ESTABLISHED -j ACCEPT
-        fi
-
-        if ! /sbin/iptables --check FORWARD -s ${SUBSPACE_IPV4_POOL} -j ACCEPT ; then
-            /sbin/iptables --append FORWARD -s ${SUBSPACE_IPV4_POOL} -j ACCEPT
-        fi
-    else 
-        echo "Unable to find /sbin/iptables not configuring IPv4 Rules"
-    fi
+if [ -n "${SUBSPACE_DISABLE_MASQUERADE-}" ]; then
```

Comment:
Only run iptables MASQUERADE when ```SUBSPACE_DISABLE_MASQUERADE``` is defined? I don't quite get this

Reply:
Fixed